packages/vkui/src/components/HorizontalCell/HorizontalCell.tsx

```diff
@@ -95,6 +79,7 @@ export const HorizontalCell = ({
   extraSubtitle,
   textAlign = size === 's' ? 'center' : 'start',
   noPadding = false,
+  titleComponent: TitleComponent = size === 's' ? Caption : Subhead,
```

Comment:
```suggestion
  TitleComponent: size === 's' ? Caption : Subhead,
```
не помню как в остальных местах, но лучше сразу через заглавную букву показывать, что компонент для конструктора

Reply:
Хотела так сделать, а в остальных местах с маленькой 😢